2016-03-18 01:51:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ð¢Ð°Ð±Ð»Ð¸Ñ†Ð° "USERS" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Table "USERS" already exists; SQL statement:
create table users (id bigint generated by default as identity, email varchar(255), login varchar(255), password varchar(255), primary key (id)) [42101-191]
2016-03-18 01:51:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: ÐžÐ³Ñ€Ð°Ð½Ð¸Ñ‡ÐµÐ½Ð¸Ðµ "UK_OW0GAN20590JRB00UPG3VA2FN" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Constraint "UK_OW0GAN20590JRB00UPG3VA2FN" already exists; SQL statement:
alter table users add constraint UK_ow0gan20590jrb00upg3va2fn  unique (login) [90045-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:98)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.CommandContainer.update(CommandContainer.java:98)
	at org.h2.command.Command.executeUpdate(Command.java:258)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:130)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:115)
	at org.hibernate.tool.hbm2ddl.DatabaseExporter.export(DatabaseExporter.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaExport.perform(SchemaExport.java:419)
	at org.hibernate.tool.hbm2ddl.SchemaExport.execute(SchemaExport.java:376)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:305)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:294)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:517)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1859)
	at com.temafon.qa.mock.service.data.DBManager.createSessionFactory(DBManager.java:55)
	at com.temafon.qa.mock.service.data.DBManager.<init>(DBManager.java:29)
	at com.temafon.qa.mock.service.data.DBManager.getInstance(DBManager.java:20)
	at com.temafon.qa.mock.service.accounts.AccountService.addNewUser(AccountService.java:32)
	at com.temafon.qa.mock.service.accounts.AccountService.<init>(AccountService.java:25)
	at com.temafon.qa.mock.service.accounts.AccountService.getInstance(AccountService.java:16)
	at verifyAuthoriztion(SignInServlet.java:115)
	at com.temafon.qa.mock.servlets.administration.AdminServlet.doGet(AdminServlet.java:18)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:735)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:848)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:648)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:455)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:229)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1072)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:382)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:193)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1006)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:135)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:116)
	at org.eclipse.jetty.server.Server.handle(Server.java:365)
	at org.eclipse.jetty.server.AbstractHttpConnection.handleRequest(AbstractHttpConnection.java:485)
	at org.eclipse.jetty.server.AbstractHttpConnection.headerComplete(AbstractHttpConnection.java:926)
	at org.eclipse.jetty.server.AbstractHttpConnection$RequestHandler.headerComplete(AbstractHttpConnection.java:988)
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:635)
	at org.eclipse.jetty.http.HttpParser.parseAvailable(HttpParser.java:235)
	at org.eclipse.jetty.server.AsyncHttpConnection.handle(AsyncHttpConnection.java:82)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint.handle(SelectChannelEndPoint.java:627)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint$1.run(SelectChannelEndPoint.java:51)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
	at java.lang.Thread.run(Thread.java:744)
2016-03-18 01:55:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Ð¢Ð°Ð±Ð»Ð¸Ñ†Ð° "USERS" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Table "USERS" already exists; SQL statement:
create table users (id bigint generated by default as identity, email varchar(255), login varchar(255), password varchar(255), primary key (id)) [42101-191]
2016-03-18 01:55:12 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: ÐžÐ³Ñ€Ð°Ð½Ð¸Ñ‡ÐµÐ½Ð¸Ðµ "UK_OW0GAN20590JRB00UPG3VA2FN" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Constraint "UK_OW0GAN20590JRB00UPG3VA2FN" already exists; SQL statement:
alter table users add constraint UK_ow0gan20590jrb00upg3va2fn  unique (login) [90045-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:98)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.CommandContainer.update(CommandContainer.java:98)
	at org.h2.command.Command.executeUpdate(Command.java:258)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:130)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:115)
	at org.hibernate.tool.hbm2ddl.DatabaseExporter.export(DatabaseExporter.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaExport.perform(SchemaExport.java:419)
	at org.hibernate.tool.hbm2ddl.SchemaExport.execute(SchemaExport.java:376)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:305)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:294)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:517)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1859)
	at com.temafon.qa.mock.service.data.DBManager.createSessionFactory(DBManager.java:55)
	at com.temafon.qa.mock.service.data.DBManager.<init>(DBManager.java:29)
	at com.temafon.qa.mock.service.data.DBManager.getInstance(DBManager.java:20)
	at com.temafon.qa.mock.service.accounts.AccountService.addNewUser(AccountService.java:32)
	at com.temafon.qa.mock.service.accounts.AccountService.<init>(AccountService.java:25)
	at com.temafon.qa.mock.service.accounts.AccountService.getInstance(AccountService.java:16)
	at verifyAuthoriztion(SignInServlet.java:122)
	at com.temafon.qa.mock.servlets.administration.AdminServlet.doGet(AdminServlet.java:18)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:735)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:848)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:648)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:455)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:229)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1072)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:382)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:193)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1006)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:135)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:116)
	at org.eclipse.jetty.server.Server.handle(Server.java:365)
	at org.eclipse.jetty.server.AbstractHttpConnection.handleRequest(AbstractHttpConnection.java:485)
	at org.eclipse.jetty.server.AbstractHttpConnection.headerComplete(AbstractHttpConnection.java:926)
	at org.eclipse.jetty.server.AbstractHttpConnection$RequestHandler.headerComplete(AbstractHttpConnection.java:988)
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:635)
	at org.eclipse.jetty.http.HttpParser.parseAvailable(HttpParser.java:235)
	at org.eclipse.jetty.server.AsyncHttpConnection.handle(AsyncHttpConnection.java:82)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint.handle(SelectChannelEndPoint.java:627)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint$1.run(SelectChannelEndPoint.java:51)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
	at java.lang.Thread.run(Thread.java:744)
2016-03-18 01:56:53 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Ð¢Ð°Ð±Ð»Ð¸Ñ†Ð° "USERS" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Table "USERS" already exists; SQL statement:
create table users (id bigint generated by default as identity, email varchar(255), login varchar(255), password varchar(255), primary key (id)) [42101-191]
2016-03-18 01:56:53 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: ÐžÐ³Ñ€Ð°Ð½Ð¸Ñ‡ÐµÐ½Ð¸Ðµ "UK_OW0GAN20590JRB00UPG3VA2FN" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Constraint "UK_OW0GAN20590JRB00UPG3VA2FN" already exists; SQL statement:
alter table users add constraint UK_ow0gan20590jrb00upg3va2fn  unique (login) [90045-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:98)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.CommandContainer.update(CommandContainer.java:98)
	at org.h2.command.Command.executeUpdate(Command.java:258)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:130)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:115)
	at org.hibernate.tool.hbm2ddl.DatabaseExporter.export(DatabaseExporter.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaExport.perform(SchemaExport.java:419)
	at org.hibernate.tool.hbm2ddl.SchemaExport.execute(SchemaExport.java:376)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:305)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:294)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:517)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1859)
	at com.temafon.qa.mock.service.data.DBManager.createSessionFactory(DBManager.java:55)
	at com.temafon.qa.mock.service.data.DBManager.<init>(DBManager.java:29)
	at com.temafon.qa.mock.service.data.DBManager.getInstance(DBManager.java:20)
	at com.temafon.qa.mock.service.accounts.AccountService.addNewUser(AccountService.java:32)
	at com.temafon.qa.mock.service.accounts.AccountService.<init>(AccountService.java:25)
	at com.temafon.qa.mock.service.accounts.AccountService.getInstance(AccountService.java:16)
	at verifyAuthoriztion(SignInServlet.java:122)
	at com.temafon.qa.mock.servlets.administration.AdminServlet.doGet(AdminServlet.java:18)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:735)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:848)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:648)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:455)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:229)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1072)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:382)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:193)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1006)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:135)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:116)
	at org.eclipse.jetty.server.Server.handle(Server.java:365)
	at org.eclipse.jetty.server.AbstractHttpConnection.handleRequest(AbstractHttpConnection.java:485)
	at org.eclipse.jetty.server.AbstractHttpConnection.headerComplete(AbstractHttpConnection.java:926)
	at org.eclipse.jetty.server.AbstractHttpConnection$RequestHandler.headerComplete(AbstractHttpConnection.java:988)
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:635)
	at org.eclipse.jetty.http.HttpParser.parseAvailable(HttpParser.java:235)
	at org.eclipse.jetty.server.AsyncHttpConnection.handle(AsyncHttpConnection.java:82)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint.handle(SelectChannelEndPoint.java:627)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint$1.run(SelectChannelEndPoint.java:51)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
	at java.lang.Thread.run(Thread.java:744)
2016-03-18 18:13:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ð¢Ð°Ð±Ð»Ð¸Ñ†Ð° "USERS" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Table "USERS" already exists; SQL statement:
create table users (id bigint generated by default as identity, email varchar(255), login varchar(255), password varchar(255), primary key (id)) [42101-191]
2016-03-18 18:13:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: ÐžÐ³Ñ€Ð°Ð½Ð¸Ñ‡ÐµÐ½Ð¸Ðµ "UK_OW0GAN20590JRB00UPG3VA2FN" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Constraint "UK_OW0GAN20590JRB00UPG3VA2FN" already exists; SQL statement:
alter table users add constraint UK_ow0gan20590jrb00upg3va2fn  unique (login) [90045-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:98)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.CommandContainer.update(CommandContainer.java:98)
	at org.h2.command.Command.executeUpdate(Command.java:258)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:130)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:115)
	at org.hibernate.tool.hbm2ddl.DatabaseExporter.export(DatabaseExporter.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaExport.perform(SchemaExport.java:419)
	at org.hibernate.tool.hbm2ddl.SchemaExport.execute(SchemaExport.java:376)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:305)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:294)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:517)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1859)
	at com.temafon.qa.mock.service.data.DBManager.createSessionFactory(DBManager.java:59)
	at com.temafon.qa.mock.service.data.DBManager.<init>(DBManager.java:30)
	at com.temafon.qa.mock.service.data.DBManager.getInstance(DBManager.java:21)
	at com.temafon.qa.mock.service.accounts.AccountService.addNewUser(AccountService.java:32)
	at com.temafon.qa.mock.service.accounts.AccountService.<init>(AccountService.java:25)
	at com.temafon.qa.mock.service.accounts.AccountService.getInstance(AccountService.java:16)
	at com.temafon.qa.mock.servlets.administration.SignInServlet.verifyAuthorization(SignInServlet.java:125)
	at com.temafon.qa.mock.servlets.administration.AdminServlet.doGet(AdminServlet.java:26)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:735)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:848)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:648)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:455)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:229)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1072)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:382)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:193)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1006)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:135)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:116)
	at org.eclipse.jetty.server.Server.handle(Server.java:365)
	at org.eclipse.jetty.server.AbstractHttpConnection.handleRequest(AbstractHttpConnection.java:485)
	at org.eclipse.jetty.server.AbstractHttpConnection.headerComplete(AbstractHttpConnection.java:926)
	at org.eclipse.jetty.server.AbstractHttpConnection$RequestHandler.headerComplete(AbstractHttpConnection.java:988)
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:635)
	at org.eclipse.jetty.http.HttpParser.parseAvailable(HttpParser.java:235)
	at org.eclipse.jetty.server.AsyncHttpConnection.handle(AsyncHttpConnection.java:82)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint.handle(SelectChannelEndPoint.java:627)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint$1.run(SelectChannelEndPoint.java:51)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
	at java.lang.Thread.run(Thread.java:744)
2016-03-18 17:16:11 database: flush
org.h2.message.DbException: Ð’Ð½ÑƒÑ‚Ñ€ÐµÐ½Ð½Ñ�Ñ� Ð¾ÑˆÐ¸Ð±ÐºÐ°: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:373)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:356)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2460)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.test(WebApp.java:893)
	at org.h2.server.web.WebApp.process(WebApp.java:221)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Ð’Ð½ÑƒÑ‚Ñ€ÐµÐ½Ð½Ñ�Ñ� Ð¾ÑˆÐ¸Ð±ÐºÐ°: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 28 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:342)
	... 23 more
2016-03-18 18:19:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ð¢Ð°Ð±Ð»Ð¸Ñ†Ð° "DYNAMIC_RESOURCES" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Table "DYNAMIC_RESOURCES" already exists; SQL statement:
create table dynamic_resources (id bigint generated by default as identity, dispatch_strategy bigint, path varchar(255), script bigint, method_id bigint, primary key (id)) [42101-191]
2016-03-18 18:19:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ð¢Ð°Ð±Ð»Ð¸Ñ†Ð° "METHODS" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Table "METHODS" already exists; SQL statement:
create table methods (id bigint generated by default as identity, name varchar(255), primary key (id)) [42101-191]
2016-03-18 18:19:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Ð¢Ð°Ð±Ð»Ð¸Ñ†Ð° "USERS" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Table "USERS" already exists; SQL statement:
create table users (id bigint generated by default as identity, email varchar(255), login varchar(255), password varchar(255), primary key (id)) [42101-191]
2016-03-18 18:19:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: ÐžÐ³Ñ€Ð°Ð½Ð¸Ñ‡ÐµÐ½Ð¸Ðµ "UK_MIST4YU9D1N9RNKKYHY9SE3WE" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Constraint "UK_MIST4YU9D1N9RNKKYHY9SE3WE" already exists; SQL statement:
alter table dynamic_resources add constraint UK_mist4yu9d1n9rnkkyhy9se3we  unique (path) [90045-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:98)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.CommandContainer.update(CommandContainer.java:98)
	at org.h2.command.Command.executeUpdate(Command.java:258)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:130)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:115)
	at org.hibernate.tool.hbm2ddl.DatabaseExporter.export(DatabaseExporter.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaExport.perform(SchemaExport.java:419)
	at org.hibernate.tool.hbm2ddl.SchemaExport.execute(SchemaExport.java:376)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:305)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:294)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:517)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1859)
	at com.temafon.qa.mock.service.data.DBManager.createSessionFactory(DBManager.java:63)
	at com.temafon.qa.mock.service.data.DBManager.<init>(DBManager.java:32)
	at com.temafon.qa.mock.service.data.DBManager.getInstance(DBManager.java:23)
	at com.temafon.qa.mock.service.accounts.AccountService.addNewUser(AccountService.java:32)
	at com.temafon.qa.mock.service.accounts.AccountService.<init>(AccountService.java:25)
	at com.temafon.qa.mock.service.accounts.AccountService.getInstance(AccountService.java:16)
	at com.temafon.qa.mock.servlets.administration.SignInServlet.verifyAuthorization(SignInServlet.java:125)
	at com.temafon.qa.mock.servlets.administration.AdminServlet.doGet(AdminServlet.java:26)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:735)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:848)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:648)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:455)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:229)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1072)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:382)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:193)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1006)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:135)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:116)
	at org.eclipse.jetty.server.Server.handle(Server.java:365)
	at org.eclipse.jetty.server.AbstractHttpConnection.handleRequest(AbstractHttpConnection.java:485)
	at org.eclipse.jetty.server.AbstractHttpConnection.headerComplete(AbstractHttpConnection.java:926)
	at org.eclipse.jetty.server.AbstractHttpConnection$RequestHandler.headerComplete(AbstractHttpConnection.java:988)
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:635)
	at org.eclipse.jetty.http.HttpParser.parseAvailable(HttpParser.java:235)
	at org.eclipse.jetty.server.AsyncHttpConnection.handle(AsyncHttpConnection.java:82)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint.handle(SelectChannelEndPoint.java:627)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint$1.run(SelectChannelEndPoint.java:51)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
	at java.lang.Thread.run(Thread.java:744)
2016-03-18 18:19:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: ÐžÐ³Ñ€Ð°Ð½Ð¸Ñ‡ÐµÐ½Ð¸Ðµ "UK_7EASYTQKF5D52YQ7X7O85PESU" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Constraint "UK_7EASYTQKF5D52YQ7X7O85PESU" already exists; SQL statement:
alter table methods add constraint UK_7easytqkf5d52yq7x7o85pesu  unique (name) [90045-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:98)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.CommandContainer.update(CommandContainer.java:98)
	at org.h2.command.Command.executeUpdate(Command.java:258)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:130)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:115)
	at org.hibernate.tool.hbm2ddl.DatabaseExporter.export(DatabaseExporter.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaExport.perform(SchemaExport.java:419)
	at org.hibernate.tool.hbm2ddl.SchemaExport.execute(SchemaExport.java:376)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:305)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:294)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:517)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1859)
	at com.temafon.qa.mock.service.data.DBManager.createSessionFactory(DBManager.java:63)
	at com.temafon.qa.mock.service.data.DBManager.<init>(DBManager.java:32)
	at com.temafon.qa.mock.service.data.DBManager.getInstance(DBManager.java:23)
	at com.temafon.qa.mock.service.accounts.AccountService.addNewUser(AccountService.java:32)
	at com.temafon.qa.mock.service.accounts.AccountService.<init>(AccountService.java:25)
	at com.temafon.qa.mock.service.accounts.AccountService.getInstance(AccountService.java:16)
	at com.temafon.qa.mock.servlets.administration.SignInServlet.verifyAuthorization(SignInServlet.java:125)
	at com.temafon.qa.mock.servlets.administration.AdminServlet.doGet(AdminServlet.java:26)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:735)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:848)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:648)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:455)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:229)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1072)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:382)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:193)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1006)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:135)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:116)
	at org.eclipse.jetty.server.Server.handle(Server.java:365)
	at org.eclipse.jetty.server.AbstractHttpConnection.handleRequest(AbstractHttpConnection.java:485)
	at org.eclipse.jetty.server.AbstractHttpConnection.headerComplete(AbstractHttpConnection.java:926)
	at org.eclipse.jetty.server.AbstractHttpConnection$RequestHandler.headerComplete(AbstractHttpConnection.java:988)
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:635)
	at org.eclipse.jetty.http.HttpParser.parseAvailable(HttpParser.java:235)
	at org.eclipse.jetty.server.AsyncHttpConnection.handle(AsyncHttpConnection.java:82)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint.handle(SelectChannelEndPoint.java:627)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint$1.run(SelectChannelEndPoint.java:51)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
	at java.lang.Thread.run(Thread.java:744)
2016-03-18 18:19:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: ÐžÐ³Ñ€Ð°Ð½Ð¸Ñ‡ÐµÐ½Ð¸Ðµ "UK_OW0GAN20590JRB00UPG3VA2FN" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Constraint "UK_OW0GAN20590JRB00UPG3VA2FN" already exists; SQL statement:
alter table users add constraint UK_ow0gan20590jrb00upg3va2fn  unique (login) [90045-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:98)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.CommandContainer.update(CommandContainer.java:98)
	at org.h2.command.Command.executeUpdate(Command.java:258)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:130)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:115)
	at org.hibernate.tool.hbm2ddl.DatabaseExporter.export(DatabaseExporter.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaExport.perform(SchemaExport.java:419)
	at org.hibernate.tool.hbm2ddl.SchemaExport.execute(SchemaExport.java:376)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:305)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:294)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:517)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1859)
	at com.temafon.qa.mock.service.data.DBManager.createSessionFactory(DBManager.java:63)
	at com.temafon.qa.mock.service.data.DBManager.<init>(DBManager.java:32)
	at com.temafon.qa.mock.service.data.DBManager.getInstance(DBManager.java:23)
	at com.temafon.qa.mock.service.accounts.AccountService.addNewUser(AccountService.java:32)
	at com.temafon.qa.mock.service.accounts.AccountService.<init>(AccountService.java:25)
	at com.temafon.qa.mock.service.accounts.AccountService.getInstance(AccountService.java:16)
	at com.temafon.qa.mock.servlets.administration.SignInServlet.verifyAuthorization(SignInServlet.java:125)
	at com.temafon.qa.mock.servlets.administration.AdminServlet.doGet(AdminServlet.java:26)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:735)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:848)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:648)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:455)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:229)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1072)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:382)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:193)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1006)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:135)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:116)
	at org.eclipse.jetty.server.Server.handle(Server.java:365)
	at org.eclipse.jetty.server.AbstractHttpConnection.handleRequest(AbstractHttpConnection.java:485)
	at org.eclipse.jetty.server.AbstractHttpConnection.headerComplete(AbstractHttpConnection.java:926)
	at org.eclipse.jetty.server.AbstractHttpConnection$RequestHandler.headerComplete(AbstractHttpConnection.java:988)
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:635)
	at org.eclipse.jetty.http.HttpParser.parseAvailable(HttpParser.java:235)
	at org.eclipse.jetty.server.AsyncHttpConnection.handle(AsyncHttpConnection.java:82)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint.handle(SelectChannelEndPoint.java:627)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint$1.run(SelectChannelEndPoint.java:51)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
	at java.lang.Thread.run(Thread.java:744)
2016-03-18 18:19:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: ÐžÐ³Ñ€Ð°Ð½Ð¸Ñ‡ÐµÐ½Ð¸Ðµ "FK_OPUENPG875F2YWGD6Q1L2KNL0" ÑƒÐ¶Ðµ Ñ�ÑƒÑ‰ÐµÑ�Ñ‚Ð²ÑƒÐµÑ‚
Constraint "FK_OPUENPG875F2YWGD6Q1L2KNL0" already exists; SQL statement:
alter table dynamic_resources add constraint FK_opuenpg875f2ywgd6q1l2knl0 foreign key (method_id) references methods [90045-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:98)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:72)
	at org.h2.command.CommandContainer.update(CommandContainer.java:98)
	at org.h2.command.Command.executeUpdate(Command.java:258)
	at org.h2.jdbc.JdbcStatement.executeUpdateInternal(JdbcStatement.java:130)
	at org.h2.jdbc.JdbcStatement.executeUpdate(JdbcStatement.java:115)
	at org.hibernate.tool.hbm2ddl.DatabaseExporter.export(DatabaseExporter.java:64)
	at org.hibernate.tool.hbm2ddl.SchemaExport.perform(SchemaExport.java:419)
	at org.hibernate.tool.hbm2ddl.SchemaExport.execute(SchemaExport.java:376)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:305)
	at org.hibernate.tool.hbm2ddl.SchemaExport.create(SchemaExport.java:294)
	at org.hibernate.internal.SessionFactoryImpl.<init>(SessionFactoryImpl.java:517)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1859)
	at com.temafon.qa.mock.service.data.DBManager.createSessionFactory(DBManager.java:63)
	at com.temafon.qa.mock.service.data.DBManager.<init>(DBManager.java:32)
	at com.temafon.qa.mock.service.data.DBManager.getInstance(DBManager.java:23)
	at com.temafon.qa.mock.service.accounts.AccountService.addNewUser(AccountService.java:32)
	at com.temafon.qa.mock.service.accounts.AccountService.<init>(AccountService.java:25)
	at com.temafon.qa.mock.service.accounts.AccountService.getInstance(AccountService.java:16)
	at com.temafon.qa.mock.servlets.administration.SignInServlet.verifyAuthorization(SignInServlet.java:125)
	at com.temafon.qa.mock.servlets.administration.AdminServlet.doGet(AdminServlet.java:26)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:735)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:848)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:648)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:455)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:229)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1072)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:382)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:193)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1006)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:135)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:116)
	at org.eclipse.jetty.server.Server.handle(Server.java:365)
	at org.eclipse.jetty.server.AbstractHttpConnection.handleRequest(AbstractHttpConnection.java:485)
	at org.eclipse.jetty.server.AbstractHttpConnection.headerComplete(AbstractHttpConnection.java:926)
	at org.eclipse.jetty.server.AbstractHttpConnection$RequestHandler.headerComplete(AbstractHttpConnection.java:988)
	at org.eclipse.jetty.http.HttpParser.parseNext(HttpParser.java:635)
	at org.eclipse.jetty.http.HttpParser.parseAvailable(HttpParser.java:235)
	at org.eclipse.jetty.server.AsyncHttpConnection.handle(AsyncHttpConnection.java:82)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint.handle(SelectChannelEndPoint.java:627)
	at org.eclipse.jetty.io.nio.SelectChannelEndPoint$1.run(SelectChannelEndPoint.java:51)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543)
	at java.lang.Thread.run(Thread.java:744)
2016-03-18 18:59:39 database: flush
org.h2.message.DbException: Ð’Ð½ÑƒÑ‚Ñ€ÐµÐ½Ð½Ñ�Ñ� Ð¾ÑˆÐ¸Ð±ÐºÐ°: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:373)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:356)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2460)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Ð’Ð½ÑƒÑ‚Ñ€ÐµÐ½Ð½Ñ�Ñ� Ð¾ÑˆÐ¸Ð±ÐºÐ°: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 28 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:342)
	... 23 more
2016-03-18 18:59:44 database: wrong user or password; user: "MOCK"
org.h2.message.DbException: Ð�ÐµÐ²ÐµÑ€Ð½Ð¾Ðµ Ð¸Ð¼Ñ� Ð¿Ð¾Ð»ÑŒÐ·Ð¾Ð²Ð°Ñ‚ÐµÐ»Ñ� Ð¸Ð»Ð¸ Ð¿Ð°Ñ€Ð¾Ð»ÑŒ
Wrong user name or password [28000-191]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.engine.Engine.openSession(Engine.java:106)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Ð�ÐµÐ²ÐµÑ€Ð½Ð¾Ðµ Ð¸Ð¼Ñ� Ð¿Ð¾Ð»ÑŒÐ·Ð¾Ð²Ð°Ñ‚ÐµÐ»Ñ� Ð¸Ð»Ð¸ Ð¿Ð°Ñ€Ð¾Ð»ÑŒ
Wrong user name or password [28000-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 19 more
2016-03-18 18:59:49 database: wrong user or password; user: "MOCK"
org.h2.message.DbException: Ð�ÐµÐ²ÐµÑ€Ð½Ð¾Ðµ Ð¸Ð¼Ñ� Ð¿Ð¾Ð»ÑŒÐ·Ð¾Ð²Ð°Ñ‚ÐµÐ»Ñ� Ð¸Ð»Ð¸ Ð¿Ð°Ñ€Ð¾Ð»ÑŒ
Wrong user name or password [28000-191]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.engine.Engine.openSession(Engine.java:106)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Ð�ÐµÐ²ÐµÑ€Ð½Ð¾Ðµ Ð¸Ð¼Ñ� Ð¿Ð¾Ð»ÑŒÐ·Ð¾Ð²Ð°Ñ‚ÐµÐ»Ñ� Ð¸Ð»Ð¸ Ð¿Ð°Ñ€Ð¾Ð»ÑŒ
Wrong user name or password [28000-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 19 more
2016-03-18 18:59:55 database: wrong user or password; user: "MOCK"
org.h2.message.DbException: Ð�ÐµÐ²ÐµÑ€Ð½Ð¾Ðµ Ð¸Ð¼Ñ� Ð¿Ð¾Ð»ÑŒÐ·Ð¾Ð²Ð°Ñ‚ÐµÐ»Ñ� Ð¸Ð»Ð¸ Ð¿Ð°Ñ€Ð¾Ð»ÑŒ
Wrong user name or password [28000-191]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.engine.Engine.openSession(Engine.java:106)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Ð�ÐµÐ²ÐµÑ€Ð½Ð¾Ðµ Ð¸Ð¼Ñ� Ð¿Ð¾Ð»ÑŒÐ·Ð¾Ð²Ð°Ñ‚ÐµÐ»Ñ� Ð¸Ð»Ð¸ Ð¿Ð°Ñ€Ð¾Ð»ÑŒ
Wrong user name or password [28000-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 19 more
2016-03-18 19:31:08 database: flush
org.h2.message.DbException: Ð’Ð½ÑƒÑ‚Ñ€ÐµÐ½Ð½Ñ�Ñ� Ð¾ÑˆÐ¸Ð±ÐºÐ°: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.get(DbException.java:168)
	at org.h2.message.DbException.convert(DbException.java:295)
	at org.h2.mvstore.db.MVTableEngine$1.uncaughtException(MVTableEngine.java:94)
	at org.h2.mvstore.MVStore.panic(MVStore.java:373)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:356)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:2888)
	at org.h2.mvstore.db.MVTableEngine$Store.open(MVTableEngine.java:154)
	at org.h2.mvstore.db.MVTableEngine.init(MVTableEngine.java:99)
	at org.h2.engine.Database.getPageStore(Database.java:2460)
	at org.h2.engine.Database.open(Database.java:692)
	at org.h2.engine.Database.openDatabase(Database.java:270)
	at org.h2.engine.Database.<init>(Database.java:264)
	at org.h2.engine.Engine.openSession(Engine.java:65)
	at org.h2.engine.Engine.openSession(Engine.java:175)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:153)
	at org.h2.engine.Engine.createSession(Engine.java:136)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:349)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:107)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Ð’Ð½ÑƒÑ‚Ñ€ÐµÐ½Ð½Ñ�Ñ� Ð¾ÑˆÐ¸Ð±ÐºÐ°: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]"
General error: "java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]" [50000-191]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 28 more
Caused by: java.lang.IllegalStateException: The file is locked: nio:C:/Users/m.guriev/IdeaProjects/mock/mockservice/h2db/mock.mv.db [1.4.191/7]
	at org.h2.mvstore.DataUtils.newIllegalStateException(DataUtils.java:773)
	at org.h2.mvstore.FileStore.open(FileStore.java:172)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:342)
	... 23 more
